scripts/performance/report-as-a-pr-comment.js

```diff
@@ -146,6 +148,22 @@ function createMessage(
   })
   message += '\n</details>\n'
 
+  message += '\n\n<details>\n<summary>🧠 Memory Performance</summary>\n\n'
+  message +=
+    '| Action Name | Base Consumption Memory (bytes) | Local Consumption Memory (bytes) | 𝚫 |\n| --- | --- | --- | --- |\n'
+  differenceMemory.forEach((memoryActionPerformance, index) => {
+    const baseMemoryPerf = memoryBasePerformance[index]
+    const localMemoryPerf = memoryLocalPerformance.find(
+      (perf) => formatActionName(perf.sdkTask) === memoryActionPerformance.name
+    )
+    const baseMemoryTaskValue = baseMemoryPerf.value !== null ? baseMemoryPerf.value : 'N/A'
+    const localMemoryTaskValue =
+      localMemoryPerf && localMemoryPerf.sdkMemoryBytes !== null ? localMemoryPerf.sdkMemoryBytes : 'N/A'
+    const diffMemoryTaskValue = memoryActionPerformance.change !== null ? memoryActionPerformance.change : 'N/A'
+    message += `| ${memoryActionPerformance.name} | ${baseMemoryTaskValue} | ${localMemoryTaskValue} | ${diffMemoryTaskValue} |\n`
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ 
```suggestion
    message += `| ${memoryActionPerformance.name} | ${formatSize(baseMemoryTaskValue)} | ${formatSize(localMemoryTaskValue)} | ${formatSize(diffMemoryTaskValue)} |\n`
```